02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Color values of 1.0 are extreme for the path tracer (these are the percent of light reflected). Let's something slightly smaller, like 0.95 so they are transferable. Maybe move the <code>fresnel.color.linear</code> up here so it is closer to the color definition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>